src/components/pages/Stories.tsx

```diff
@@ -66,6 +66,10 @@ export default function Stories() {
 
   const setIndex = React.useCallback((index: number) => dispatch(setFullscreenPost(index)), [dispatch])
 
+  const scrollToPrevious = React.useCallback(() => {
+    document.getElementById(`story-${prevIdx}`)?.scrollIntoView({ behavior: 'smooth' })
+  }, [nextIdx])
```

Comment:
Probably a typo here. Should be `prevIdx` right?

Reply:
You got me. I just mindlessly copied the scrollToNext function.